# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/svg/SVGIntegrationUtils.h
# Commit: 0fdbf0a455f2
# Full Hash: 0fdbf0a455f26afa0a7a0390d6a380e318fce9d4
# Author: Robert Longson <longsonr@gmail.com>
# Date: 2023-10-24 03:54:44
# Regressor Bug: 1765202
# File Overlap Count: 1
# Description:
#   Bug 1765202 part 3 - ignore clip-path and mask properties that don't point to clipPath and mask elements r=emilio
#   
#   This refactors the logic for testing whether something has a clip-path or mask and what to do about it in one place rather than using many overlapping functions.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D190767
# ==============================================================================

diff -r 2a8d11fcc746 -r 0fdbf0a455f2 layout/svg/SVGIntegrationUtils.h
--- a/layout/svg/SVGIntegrationUtils.h	Mon Oct 23 20:32:20 2023 +0000
+++ b/layout/svg/SVGIntegrationUtils.h	Mon Oct 23 20:32:37 2023 +0000
@@ -61,17 +61,6 @@
   static bool UsingEffectsForFrame(const nsIFrame* aFrame);
 
   /**
-   * Returns true if mask or clippath are currently applied to this frame.
-   */
-  static bool UsingMaskOrClipPathForFrame(const nsIFrame* aFrame);
-
-  /**
-   * Returns true if the element has a clippath that is simple enough to
-   * be represented without a mask in WebRender.
-   */
-  static bool UsingSimpleClipPathForFrame(const nsIFrame* aFrame);
-
-  /**
    * Returns the size of the union of the border-box rects of all of
    * aNonSVGFrame's continuations.
    */